docs/source/daml-script/template-root/daml.yaml.template

```diff
@@ -0,0 +1,16 @@
+sdk-version: __VERSION__
+name: script-example
+source: src
+parties:
+  - Alice
+  - Bob
+  - Bank
+version: 0.0.1
+# script-dependencies-begin
+dependencies:
+  - daml-prim
+  - daml-stdlib
+  - daml-script
+# script-dependencies-end
+build-options:
+  - --target=1.7
```

Comment:
We can now remove this. :)

Reply:
We can’t, the corresponding PR is still failing on a bunch of tests. Working on fixing that atm :slightly_smiling_face: